### DISCUSSION TITLE
Font size crashes MPF-MC

### DISCUSSION THREAD

**Discussion by MaxPowerTech** [missionpinball/missionpinball#90] [Q&A]
Started a project on an older version of MPF and trying to get it running on a new 0.56 install.  MPF-MC was silently crashing without producing an error.

Traced it down to the title text on the welcome slide.  Using the Sinbad font (https://fontsgeek.com/sinbad-font) it will crash mpf-mc if the size is greater than 212, but if I change the font (ex Impact) it will not crash.  The previous version of MPF had no problem with this:

slides:
  welcome_slide:
    widgets:
      - type: text
        text: Sinbad
        font_size: 500
        font_name: Sinbad
        color: yellow
        y: center

**Comment by cobra18t** [comment]
The MC is a mystery to me. I just try to make it happy and back slowly away.

If you need something that big in that font, could you reduce the resolution of your display window? You could use the 212 font size and the display driver will upscale to fill the window as necessary.

**Reply by MaxPowerTech** [reply]
My game is fixed at 1080p for the video content.

There are other slides where a font size of less than 212 will crash MC, guessing it has to do with the rendering area.

If I really need huge static text I'll workaround the issue by making an image file with it.
